src/rest_api/classes/laboratory/widgets/alleles.clj

```diff
@@ -0,0 +1,42 @@
+(ns rest-api.classes.laboratory.widgets.alleles
+  (:require
+    [datomic.api :as d]
+    [clojure.string :as str]
+    [pseudoace.utils :as pace-utils]
+    [rest-api.formatters.object :as obj :refer [pack-obj]]))
+
+(defn alleles [laboratory]
+  (let [db (d/entity-db laboratory)
+        data (->> (d/q '[:find [?varent ...]
+                         :in $ ?laboratory
+                         :where 
+                         [?laboratory :laboratory/alleles ?varent] ]
+                       db (:db/id laboratory))
+                  (map (fn [wbvar]
+                         (let [allele (d/entity db wbvar)]
+                           (pace-utils/vmap
+                             :sequenced 
+                             (if (= (str (:variation/seqstatus allele))
```

Comment:
No need to cast to `str` here for equality testing, can simply do:
```clojure
(if (= (:variation/seqstatus allele) :variation.seqstatus/sequenced)
   "yes"
   "no")
```

Reply:
Ah, cool, thanks.